Automattic/woocommerce-payments
# Fix: added missing display attribute to the table row data

Fixes #1891 

#### Changes proposed in this Pull Request
- added missing display attribute to the table row data.

#### Testing instructions
- checkout to `fix/1891-fix-transaction-id`.
- goto transactions list page.
- toggle show transaction id from the three-dot menu.
- Transaction id should be visible now.

#### After fix
![image](https://user-images.githubusercontent.com/15019298/119499526-6ef7d300-bd84-11eb-9009-200e39e5c28e.png)

-------------------
